CMakeLists.txt

```diff
@@ -548,6 +549,20 @@ if(AVIF_BUILD_TESTS)
     endif()
     target_link_libraries(aviftest avif ${AVIF_PLATFORM_LIBRARIES})
     target_include_directories(aviftest PRIVATE apps/shared)
+    add_custom_target(avif_test_all
+        COMMAND $<TARGET_FILE:aviftest> ${CMAKE_CURRENT_SOURCE_DIR}/tests/data
+        DEPENDS aviftest
+    )
```

Comment:
1. I added `enable_testing()` so that the `ctest` command will run all targets referenced by `add_test()`. \
I moved the `add_custom_target(avif_test_all ...)` call closer to its only dependency `aviftest`. \
These two changes are not required but `ctest` is convenient and the move seemed clearer to me.
2. I do not mind adding `avifgridapitest` and `avifyuv` to the `avif_test_all` custom target but this seems beyond the scope of this change. Is `avif_test_all` used by the CI of this GitHub repo?

Reply:
I put `add_custom_target()` back to where it was.